FirebaseAuth/Sources/Backend/RPC/FIREmailLinkSignInResponse.h

```diff
@@ -49,6 +55,10 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic, assign) BOOL isNewUser;
 
+@property(nonatomic, strong, readonly, nullable) NSString *MFAPendingCredential;
+
+@property(nonatomic, strong, readonly, nullable) NSArray<FIRAuthProtoMFAEnrollment *> *MFAInfo;
```

Comment:
nit: Can you add this documentation:

```
/** @property MFAInfo
    @brief Info on which multi-factor authentication providers are enabled.
*/
```

Reply:
Yes, I can. Should I add same comment in `FIRVerifyPasswordResponse`? [here](https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseAuth/Sources/Backend/RPC/FIRVerifyPasswordResponse.h#L71)

Reply:
Yes, and in `FIRVerifyAssertionResponse`: https://github.com/firebase/firebase-ios-sdk/blob/3cb4d9ceae4bda421e5a7ebd050c64192b523515/FirebaseAuth/Sources/Backend/RPC/FIRVerifyAssertionResponse.h#L210 Thank you!